### ISSUE TITLE
ambiguity warnings

### ISSUE THREAD

**Issue by tbreloff** [timholy/Images.jl#417]
I've been getting a lot of these for a while:

```
WARNING: New definition 
    ./(AbstractArray, Union{DataArrays.DataArray, DataArrays.PooledDataArray}) at /home/tom/.julia/v0.4/DataArrays/src/broadcast.jl:285
is ambiguous with: 
    ./(Images.AbstractImageDirect, AbstractArray) at /home/tom/.julia/v0.4/Images/src/algorithms.jl:55.
To fix, define 
    ./(Images.AbstractImageDirect, Union{DataArrays.DataArray, DataArrays.PooledDataArray})
before the new definition.
```

**Comment by timholy** [comment]
https://github.com/JuliaStats/DataArrays.jl/issues/168

**Comment by tbreloff** [comment]
Thanks for the link, and sorry about the noise.

**Comment by timholy** [comment]
No prob.

**Comment by crbinz** [comment]
Sorry about dredging this up, but if I follow the chain of issues/PRs, I think I'm seeing that this is a known issue that will not be fixed for v0.4, is that right?
